ethereum/helpers/uris.ts

```diff
@@ -1,72 +1,89 @@
-export function getURITemplate(
+import { init, __wasm } from "@tableland/sqlparser";
+
+export async function normalize(sql: string) {
+  if (__wasm == null) {
+    await init();
+  }
+  return (await globalThis.sqlparser.normalize(sql)).statements[0];
+}
+
+export async function getURITemplate(
   tablelandHost: string,
   attributesTable: string,
   lookupsTable: string,
   pilotsTable: string,
   displayAttributes: boolean
-): string[] {
+): Promise<string[]> {
   if (!displayAttributes) {
     const uri =
       tablelandHost +
-      "/query?extract=true&unwrap=true&s=" +
+      "/api/v1/query?format=objects&extract=true&unwrap=true&statement=" +
```

Comment:
Updated to the new API url. cc @joewagner

Reply:
I think `format=objects` is the default, but added it just to be explicit and future proof since this ends up in smart contract storage.

Reply:
Could we use the sdk somehow to build this uri instead so that we don't have to remember to make updates here if the api changes? Might not be worth it since it probably won't change very often

Reply:
Took a quick look and I don't see anything useful from the sdk here.

Reply:
Sorry, I'm a little slow to respond here.  The way you did this works, so no need to change now, but you can potentially leverage the sdk's helpers if you know the chain being targeted.
example:
```
import { helpers } from "@tableland/sdk";

const tablelandHost = helpers.getChainInfo("homestead").baseUrl;
```